vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/Sequence.java

```diff
@@ -31,6 +33,8 @@
     @OneToOne(targetEntity = Module.class)
     private IModule module;
 
+    //-------- @JsonIgnore used as this Slide will be returned in a controller
```

Comment:
I'm pretty sure that you need this in the controller?

Reply:
Do I need to change?